README.md

```diff
@@ -232,6 +232,18 @@ If you have custom modules you can mount them to the following directory:
 
 - **/var/www/html/modules/name-of-module**
 
+**PHP Configuration**
```

Comment:
This gives guidance for PHP.ini and nginx conf.d
 = significant for dev/test, also per strategies for moderating config injection if refactoring CDK to target container deployment onto EC2
 = GREAT!
 
However, in the wild, a prime use case for this is tuning page timeout for sites with known bottlenecks.
Unfortunately the paths cited here are insufficient for complete config (of timeout) vs permutations of nginx setup.
Also needed is influence over : /etc/php/8.1/fpm/pool.d/www.conf
Which the current CDK does not provision!

Can you add that path here also, to help preserve intelligence pending step-wise improvement of CDK capacity?

Reply:
(/etc/php/8.1/fpm/pool.d/www.conf ... noting, that is vanilla Ubuntu, Alpine path may resolve differently!)

Reply:
Added some more info